rheem-platforms/rheem-flink/src/main/java/org/qcri/rheem/flink/compiler/criterion/RheemConvergenceCriterion.java

```diff
@@ -0,0 +1,45 @@
+package org.qcri.rheem.flink.compiler.criterion;
```

Comment:
Same comment holds from above re-class naming. Also, some java docs would be nice